README.md

```diff
@@ -354,25 +354,32 @@ easier.
 
   * `--no-pstate-check`: Do not crash out if Intel P-states are not disabled.
 
-  * `--hardware-reboots`: Restart physical hardware before each benchmark
-    execution.
-
 Note that you should not collect results intended for publication with
 development switches turned on.
 
+We also recommend that for 'real' benchmarking you turn on the `--hardware-reboots`
```

Comment:
We were going to move this explanation out of the `Development and Debug Switches` section, weren't we? I'd recommend moving it to where the user is running the example scenario.